.travis.yml

```diff
@@ -70,6 +73,8 @@ matrix:
       env: PRESET=CC3200_SDK120 XI_MAKETARGET=""
     - compiler: clang
       env: PRESET=CC3200_TLS_SOCKET XI_MAKETARGET=""
+    - compiler: clang
```

Comment:
This is a Travis CI config file. I may not understand this file well enough but have a certain level of understanding: there are two compilers running through the build configs: clang and gcc. Non of these are cross-compilers. Maybe Travis isn't for cross compilations? Don't know but we use gcc run to do cross-compilations. The clang run cross-compilations are disabled since it would be the same as the gcc since the compiler is replaced anyways. This two-liner if for disabling ESP32 build for the clang run. And actually you can see the other cross-builds here too.
Not the prettiest...but it is enough for us for now.

Reply:
Ahhh got it, thanks :) Didn't see the `exclude:` line.